.github/workflows/build.yml

```diff
@@ -18,7 +18,9 @@ on:
 
 env:
   DOTNET_VERSION: ${{ '8.0.201' }}
-  ENABLE_DIAGNOSTICS: true
+  # Debug logging enabled via GitHub UI during reruns or set as repo secret.
+  # See also https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
+  ENABLE_DIAGNOSTICS: ${{ github.ACTIONS_RUNNER_DEBUG }}
```

Comment:
Hmm, doesn't seem to be picking it up, should it just be a straight `ACTIONS_RUNNER_DEBUG ` or `secrets.ACTIONS_RUNNER_DEBUG `?

Reply:
Oh, just found this, it may just be `${{ runner.debug }}` https://stackoverflow.com/questions/75434105/how-to-determine-if-the-github-action-is-running-with-debug-logging-enabled

From runner context doc - https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context - this seems to be what corresponds to the checkbox. Not sure what that other doc is about...

We should update the comment too

Reply:
Looking at the originally linked docs, I think I see why `ACTIONS_RUNNER_DEBUG` didn't enable verbose logging. 
- The runner process log, which includes information about coordinating and setting up runners to execute jobs.
- The worker process log, which logs the execution of a job.

Maybe we need to enable _step_ debug logging instead via `ACTIONS_STEP_DEBUG`, which:
- increases the verbosity of a job's logs during and after a job's execution.

As for the `runner.debug` context value, the docs for this seem to point to the same docs used before 🤔

We can try both, but let's start with `ACTIONS_STEP_DEBUG` and see if that works.

Reply:
```suggestion
  ENABLE_DIAGNOSTICS: ${{ github.ACTIONS_STEP_DEBUG }}
```

Reply:
Looks like `ACTIONS_STEP_DEBUG` didn't do it either, let's try `runner.debug` then

Reply:
Posted [here](https://github.com/CommunityToolkit/Windows/pull/453#issuecomment-2253238146) as well, but `runner.debug` didn't work as expected. `runner` isn't recognized.